data/blog/2024-12-16-fall-2024-engr1a-cyber-lab.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Mastering Offensive Security, Building Our Own Modular Malware
+authors: [Samyukhtha Rajkumar Sridevi]
+category: Projects
+tags: [fall-2024, engr1a-cyber-lab]
+description: Building our own modular malware
+---
+
+# Mastering Offensive Security: Building Our Own Modular Malware
+## Introduction
+For our final project in the ENGR 1A Cyber Lab, my group and I designed and implemented a complete offensive security framework. Combining key concepts from the course (such as enumeration, exploitation, privilege escalation, persistence, and command-and-control (C&C)) we built our own modular malware. We used an attack virtual machine to attack and exploit a target virtual machine. Each stage taught us something new about offensive strategies, system vulnerabilities, and practical cybersecurity applications.
```

Comment:
Also on me for not telling you, but could the markdown be formatted with one sentence = one line? Thank you!